aiida/orm/querybuilder.py

```diff
@@ -1994,15 +1995,20 @@ def get_query(self):
                 self._hash = queryhelp_hash
         return query
 
-    @staticmethod
-    def get_aiida_entity_res(value):
+    def get_aiida_entity_res(self, value):
```

Comment:
could be a `@classmethod`, right?

Reply:
could be, but I think staticmethod makes more sense. Now that I fixed the problem in the correct place (thanks for catching this :+1:) I think we can keep this as was